grapple/types/pages.py

```diff
@@ -191,10 +196,13 @@ class Mixin:
             content_type=graphene.String(),
         )
 
-        # Return all pages, ideally specific.
+        # Return all pages in site, ideally specific.
         def resolve_pages(self, info, **kwargs):
+            site = Site.find_for_request(info.context)
             return resolve_queryset(
-                WagtailPage.objects.live().public().specific(), info, **kwargs
+                WagtailPage.objects.in_site(site).live().public().specific(),
```

Comment:
This is a breaking change.
We may add an argument like `in_site=graphene.Boolean()` and when `True` we would filter pages with `.in_site(site)`.
This way, we add the feature and keep backward compatibility 🙂.

Reply:
This was based on my note from https://github.com/GrappleGQL/wagtail-grapple/issues/114#issuecomment-694905294

The REST API uses `in_site` as a filter. I have since seen a few queries about multi-site via a single endpoint which makes me think the site filtering should be opt-in. So good shout @ruisaraiva19 !